.github/workflows/clang-ast.yaml

```diff
@@ -0,0 +1,50 @@
+name: "Clang AST"
+
+on:
+  push:
+    branches: master
+    paths:
+      - configure
+      - 'auto/**'
+      - 'src/**'
+      - 'test/**'
+      - '.github/workflows/clang-ast.yaml'
+  pull_request:
+    branches: master
+    paths:
+      - configure
+      - 'auto/**'
+      - 'src/**'
+      - 'test/**'
+      - '.github/workflows/clang-ast.yaml'
+
+jobs:
+  clang-ast:
+    runs-on: ubuntu-latest
+
+    container:
+      image: debian:testing
+
+    steps:
+      - name: Install tools/deps
+        run: |
+          apt-get -y update
+          apt-get -y install git llvm-dev libclang-dev clang make \
+                             libssl-dev libpcre2-dev
+
+      - uses: actions/checkout@v4
+
+      - name: Checkout and build clang-ast
+        run: |
+          git clone https://github.com/nginx/clang-ast -b unit
+          cd clang-ast
+          make
+
+      - name: Configure Unit
+        run: ./configure --cc=clang --openssl --debug
+
+      - name: Build Unit
+        run: |
+          make -j4 \
+            EXTRA_CFLAGS="-Xclang -load -Xclang clang-ast/ngx-ast.so -Xclang -plugin -Xclang ngx-ast" \
```

Comment:
Should this be split across multiple lines to adhere to our repository wide 80 character line length limit?

Reply:
No, I don't think so.

As you can see, I've split long lines where it makes sense.

This is no different to the other workflow files or the README for example.